app/server.go

```diff
@@ -579,8 +579,8 @@ func (s *Server) startInterClusterServices(license *model.License) error {
 
 	// Remote Cluster service
 
-	// License check
-	if !*license.Features.RemoteClusterService {
+	// License check  (assumed enabled if shared channels enabled)
```

Comment:
Super nit: I think there is an extra space between `check` and the parenthesis.

Reply:
Fixed. Thanks.